lms/djangoapps/experiments/utils.py

```diff
@@ -238,29 +220,32 @@ def get_experiment_user_metadata_context(course, user):
                 courses_left_to_purchase_price = None
                 courses_left_to_purchase_url = None
                 program_uuid = program.get('uuid')
+                is_eligible_for_one_click_purchase = program.get('is_program_eligible_for_one_click_purchase')
                 if courses is not None:
                     total_courses = len(courses)
                     complete_enrollment = is_enrolled_in_all_courses(courses, user_enrollments)
 
-                    if PROGRAM_PRICE_FLAG.is_enabled():
-                        # Get the price and purchase URL of the program courses the user has yet to purchase. Say a
-                        # program has 3 courses (A, B and C), and the user previously purchased a certificate for A.
-                        # The user is enrolled in audit mode for B. The "left to purchase price" should be the price of
-                        # B+C.
-                        non_audit_enrollments = [enrollment for enrollment in user_enrollments if enrollment not in
-                                                 audit_enrollments]
-                        courses_left_to_purchase = get_unenrolled_courses(courses, non_audit_enrollments)
-                        if courses_left_to_purchase:
-                            has_courses_left_to_purchase = True
-                        courses_left_to_purchase_price, courses_left_to_purchase_skus = get_program_price_and_skus(
-                            courses_left_to_purchase)
-                        courses_left_to_purchase_url = EcommerceService().get_checkout_page_url(
-                            *courses_left_to_purchase_skus, program_uuid=program_uuid)
+                    # Get the price and purchase URL of the program courses the user has yet to purchase. Say a
+                    # program has 3 courses (A, B and C), and the user previously purchased a certificate for A.
+                    # The user is enrolled in audit mode for B. The "left to purchase price" should be the price of
+                    # B+C.
+                    non_audit_enrollments = [enrollment for enrollment in user_enrollments if enrollment not in
+                                             audit_enrollments]
+                    courses_left_to_purchase = get_unenrolled_courses(courses, non_audit_enrollments)
```

Comment:
This is not a question about the code, but rather how courses/enrollments work: if you are enrolled in a course, but you are not auditing it, doesn't that imply you have already purchased it? What does it mean if you are not auditing, but you also have not purchased it? What is the name of this state?